src/System.Windows.Forms/src/System/Windows/Forms/ListViewGroupEventArgs.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Windows.Forms
+{
+    /// <summary>
+    ///  Provides index of the group for the <see cref='ListView.OnGroupTaskLinkClick'/> event.
```

Comment:
Here and below - all members must be documented, and have consistent wording. Please look at [`ListViewItemMouseHoverEventArgs`](https://docs.microsoft.com/dotnet/api/system.windows.forms.listviewitemmousehovereventargs) or any other existing doc for inspiration.

```suggestion
    ///  Provides data for the <see cref='ListView.OnGroupTaskLinkClick'/> event.
```